fs-diff-test-utils/diff-fs.sh

```diff
@@ -0,0 +1,7 @@
+result=`podman diff $1 | cut -d " " -f 2 | grep -E "(^/var/lib/rpm)|(^/var/lib/dpkg)|(^/bin)|(^/sbin)|(^/lib)|(^/lib64)|(^/usr/bin)|(^/usr/sbin)|(^/usr/lib)|(^/usr/lib64)"`
+if [ -z "${result}" ]
+then
+echo empty
+else
+echo $result
+fi
```

Comment:
Should be implemented using the `reel.Handler` FSM and `tnf.Test` interface per https://github.com/test-network-function/test-network-function/blob/main/CONTRIBUTING.md#test-implementation-guidelines.

Reply:
Per our earlier discussion, this is code running on an injected Pod, and thus is fine to be bash code.  However, the code that invokes this command should be a Handler still.